### PR TITLE
feat(css): Add CSS Values Level 4 `atan2()` compatibility data

### PR DESCRIPTION
**Pull request by ramiy**
#### Summary
Document the CSS `atan2()` functional notation.

#### Test results and supporting details
[Safari/webkit already supports](https://webkit.org/blog/12445/new-webkit-features-in-safari-15-4/) this CSS function. [Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=1190444) will follow soon.

### CODE REVIEW

File: css/types/atan2.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
```
Comment (queengooborg) [inline css/types/atan2.json:18]:
```suggestion
            "chrome_android": "mirror",
            "edge": "mirror",
```

File: css/types/atan2.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
```
Comment (queengooborg) [inline css/types/atan2.json:32]:
```suggestion
            "opera": "mirror",
            "opera_android": "mirror",
```

File: css/types/atan2.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            "safari": {
+              "version_added": "15.4"
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": {
+              "version_added": false
+            }
```
Comment (queengooborg) [inline css/types/atan2.json:40]:
```suggestion
            "webview_android": "mirror"
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
LGTM!  Let's pre-set Chromium derivatives as "mirror" as well while we're here for easier maintenance down the line.  (This applies to all three of your most recent PRs.)

**Comment by ramiy** [comment]
Hi @queengooborg, I updated all my 7 PRs. Please review and merge.

**Review by queengooborg — APPROVED** [review]
Thank you for the quick changes!

**Pull request merged**